src/MIDebugEngine/AD7.Impl/AD7MemoryAddress.cs

```diff
@@ -160,19 +158,15 @@ public int GetInfo(enum_CONTEXT_INFO_FIELDS dwFields, CONTEXT_INFO[] pinfo)
             {
                 pinfo[0].dwFields = 0;
 
-                if ((dwFields & enum_CONTEXT_INFO_FIELDS.CIF_ADDRESS) != 0)
+                if ((dwFields & enum_CONTEXT_INFO_FIELDS.CIF_ADDRESS) != 0 ||
+                    (dwFields & enum_CONTEXT_INFO_FIELDS.CIF_ADDRESSABSOLUTE) != 0)
                 {
                     pinfo[0].bstrAddress = EngineUtils.AsAddr(_address, _engine.DebuggedProcess.Is64BitArch);
-                    pinfo[0].dwFields |= enum_CONTEXT_INFO_FIELDS.CIF_ADDRESS;
+                    pinfo[0].bstrAddressAbsolute = pinfo[0].bstrAddress;
```

Comment:
>bstrAddressAbsolute [](start = 29, length = 19)

You don't want to assign to these fields unless they are requested. This can cause a memory leak in the Visual Studio case.